osu.Game/Screens/OnlinePlay/Header.cs

```diff
@@ -125,35 +99,5 @@ private void load(OsuColour colours)
                 pageTitle.Colour = dot.Colour = colours.Yellow;
             }
         }
-
-        private class HeaderBreadcrumbControl : ScreenBreadcrumbControl
```

Comment:
The test failures is due to removal of this component (the header items are no longer clickable). The replacement looks suspiciously like a breadcrumb also, however:

![2021-08-13-000206_331x79_scrot](https://user-images.githubusercontent.com/20418176/129275712-5450369f-4351-4f27-92ea-88836bd69aa3.png)

Unsure if it was intended to be interactable?

Reply:
Fixed the test.

> Unsure if it was intended to be interactable

Not sure. The design hasn't changed from master, so I didn't assume so.